prism_benchmarks/data/parser/sorbet-tests/2024-07-18-f75db5824.json

```diff
@@ -0,0 +1,78 @@
+{
+  "results": [
+    {
+      "command": "bazel-bin/main/sorbet --parser=sorbet --stop-after=parser test/testdata",
+      "mean": 10.0427811339,
+      "stddev": 0.025126222846626647,
+      "median": 10.0368188785,
+      "user": 15.975400439999998,
+      "system": 1.0591082200000002,
+      "min": 10.0078213055,
+      "max": 10.0817527725,
+      "times": [
+        10.0276461275,
+        10.0627562915,
+        10.0649185675,
+        10.0174194365,
+        10.0078213055,
+        10.0817527725,
+        10.0454906105,
+        10.0242698005,
+        10.0281471465,
+        10.0675892805
+      ],
+      "exit_codes": [
+        100,
+        100,
+        100,
+        100,
+        100,
+        100,
+        100,
+        100,
+        100,
+        100
```

Comment:
This is something to explore further -- Prism encountering parsing errors in the Sorbet tests that it doesn't know how to recover from.